devdoc/DockerImage.md

```diff
@@ -0,0 +1,45 @@
+# Docker Images
+
+_This document describes how to create and use a docker image for the HPCC-Platform._
+
+## Creating a Docker Image
+
+The HPCC-Platform repository contains a Dockerfile that can be used to create a docker image for the HPCC-Platform including all the required dependencies for a community edition.  The Dockerfile is located in [dockerfiles/vcpkg/platform-core-ubuntu-22.04](../dockerfiles/vcpkg/platform-core-ubuntu-22.04/Dockerfile) and is named `Dockerfile`.  The image can then be used to run the HPCC-Platform in a docker container.
+
+### tldr;
+
+_Too long; didn't read_
+
+```sh
+mkdir tmp
+cd tmp
+wget https://raw.githubusercontent.com/hpcc-systems/HPCC-Platform/master/dockerfiles/vcpkg/platform-core-ubuntu-22.04/Dockerfile
+wget https://github.com/hpcc-systems/HPCC-Platform/releases/download/community_9.8.10-1/hpccsystems-platform-community_9.8.10-1jammy_amd64_withsymbols.deb
+docker build --build-arg PKG_FILE=./hpccsystems-platform-community_9.8.10-1jammy_amd64_withsymbols.deb -t my-platform-image .
```

Comment:
I get an error:

ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref moby::q5y85slgq163a94savpcawciw: "/home/gavin/buildcontainer/hpccsystems-platform-community_9.6.37-closedown0Debugjammy_amd64_withsymbols_k8s.deb": not found

That file does exist.  I am not using docker desktop.

Reply:
I just tested here and it worked:
```sh
 => [internal] load build definition from Dockerfile                                                                                                                                0.0s
 => => transferring dockerfile: 2.89kB                                                                                                                                              0.0s
 => [internal] load metadata for docker.io/library/ubuntu:22.04                                                                                                                     4.7s
 => [internal] load .dockerignore                                                                                                                                                   0.0s
 => => transferring context: 2B                                                                                                                                                     0.0s
 => [ 1/17] FROM docker.io/library/ubuntu:22.04@sha256:adbb90115a21969d2fe6fa7f9af4253e16d45f8d4c1e930182610c4731962658                                                             6.5s
 => => resolve docker.io/library/ubuntu:22.04@sha256:adbb90115a21969d2fe6fa7f9af4253e16d45f8d4c1e930182610c4731962658                                                               0.0s
 => => sha256:857cc8cb19c0f475256df4b7709003b77f101215ebf3693118e61aac6a5ea4ff 29.54MB / 29.54MB                                                                                    5.5s
 => => extracting sha256:857cc8cb19c0f475256df4b7709003b77f101215ebf3693118e61aac6a5ea4ff                                                                                           0.9s
 => [internal] load build context                                                                                                                                                   2.4s
 => => transferring context: 350.15MB                                                                                                                                               2.3s
 => [ 2/17] RUN apt-get clean -y &&     apt-get autoclean -y &&     apt-get install -y -f &&     apt-get autoremove -y &&     apt-get update -y &&     apt-get install --no-inst  111.8s
 => [ 3/17] RUN if [ 1 -eq 1 ] ; then apt-get install -y libcppunit-1.15-0 ; fi                                                                                                     3.0s
 => [ 4/17] RUN apt-get install -y     dnsutils     gdb     nano                                                                                                                    8.5s
 => [ 5/17] RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.29.7/bin/linux/amd64/kubectl &&     chmod +x ./kubectl &&     mv ./kubectl /usr/local/bin    10.8s
 => [ 6/17] RUN curl -LO https://packagecloud.io/github/git-lfs/packages/ubuntu/jammy/git-lfs_3.5.1_amd64.deb/download &&     dpkg -i download &&     rm download                   3.6s
 => [ 7/17] RUN locale-gen en_US.UTF-8                                                                                                                                              2.0s
 => [ 8/17] RUN groupadd -g 10001 hpcc                                                                                                                                              0.7s
 => [ 9/17] RUN useradd -s /bin/bash -m -r -N -c "hpcc runtime User" -u 10000 -g hpcc hpcc                                                                                          0.9s
 => [10/17] RUN passwd -l hpcc                                                                                                                                                      0.6s
 => [11/17] RUN mkdir /var/lib/HPCCSystems && chown hpcc:hpcc /var/lib/HPCCSystems                                                                                                  0.6s
 => [12/17] RUN mkdir /var/log/HPCCSystems && chown hpcc:hpcc /var/log/HPCCSystems                                                                                                  0.6s
 => [13/17] RUN mkdir /var/lock/HPCCSystems && chown hpcc:hpcc /var/lock/HPCCSystems                                                                                                0.8s
 => [14/17] RUN mkdir /var/run/HPCCSystems && chown hpcc:hpcc /var/run/HPCCSystems                                                                                                  0.7s
 => [15/17] COPY ././hpccsystems-platform-community_9.8.10-1jammy_amd64_withsymbols.deb /tmp/./hpccsystems-platform-community_9.8.10-1jammy_amd64_withsymbols.deb                   1.0s
 => [16/17] RUN dpkg -i /tmp/./hpccsystems-platform-community_9.8.10-1jammy_amd64_withsymbols.deb &&     apt-get install -f &&     rm /tmp/./hpccsystems-platform-community_9.8.1  11.5s
 => [17/17] WORKDIR /var/lib/HPCCSystems                                                                                                                                            0.2s
 => exporting to image                                                                                                                                                             84.5s
 => => exporting layers                                                                                                                                                            62.0s
 => => exporting manifest sha256:425024184723d388f03ffbb29b72995b66f5e19089b77a46f03224e19f019daa                                                                                   0.0s
 => => exporting config sha256:cb551f113a76a921ba03b411fc6cd426e0f841917f957af1bcee246ae85767f9                                                                                     0.0s
 => => exporting attestation manifest sha256:72945a04cb0339c4d0c2f326ee2a867ae12b259890d6879b0dd7496e1d296f0a                                                                       0.0s
 => => exporting manifest list sha256:6ebe82261596ac3bda2bb5da25ce8844ecefa356f26f75232fb614dabcfffe1f                                                                              0.0s
 => => naming to docker.io/library/my-platform-image:latest                                                                                                                         0.0s
 => => unpacking to docker.io/library/my-platform-image:latest  
```

Reply:
Did you include the `.` at the end of the line?